### ISSUE TITLE
Revise screenshots

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/ph-scale#256]
Noted after creating a dev version for https://github.com/phetsims/ph-scale/issues/236 ...

The .png files published with the sim do not match current reality. For example:

ph-scale-io.png:

![ph-scale-ios](https://user-images.githubusercontent.com/3046552/207914187-72e51575-fe64-4e72-b45e-9d87a7f40c20.png)

actual sim:

<img width="535" alt="screenshot_2068" src="https://user-images.githubusercontent.com/3046552/207914238-9cba0688-dafb-416c-880b-e3770a200efc.png">

Note the navigation bar. The Home button is in a different place, and a different size. The Preferences button is missing.

These .png files are generated from assets/*screenshots.png.  So I'm guessing that all of the screenshots in ph-scale/assets/ and ph-scale-basics/assets need a refresh.

@arouinfar can you do this?

**Comment by pixelzoom** [comment]
I guess another questions is:  Is it worth the effort to revise the screenshots, or are the current screenshots good enough?

**Comment by pixelzoom** [comment]
> The Preferences button is missing.

The Preferences buttons is only present when running the all.html file, so it probably should not be in the screenshots. But I'm not sure...

**Comment by arouinfar** [comment]
We don't necessarily update sim screenshots when the only minor changes are to the navbar (like adding a single button). However, there have been lots of changes to the navbar: menu dots, home button size and placement, preferences button, and the size of the PhET logo. I think it's worthwhile to update the screenshots, so I'll take care of that.

> The Preferences button is only present when running the all.html file (ph-scale_all_phet.html, ph-scale-basics_all_phet.html), so it probably should not be in the screenshots. But I'm not sure...

That's an interesting point, and I don't think we have an established pattern here. Since the goal is to eventually serve the _all file on the sim page, I think it's okay for the Preference button to be visible in the screenshots.

**Comment by pixelzoom** [comment]
So.... Since it sounds like we want to update the screenshots for 1.6 publication, I'll label this as blocks-sim-publication.

**Comment by arouinfar** [comment]
Screenshots for ph-scale and ph-scale-basics updated in the above commits. I created an alt3 for ph-scale because the previous screenshots used for the website did not feature the Macro screen at all.  I took the screenshots from master to pick up the dashed line in the pH meter #262, but forgot to use `locales` so there's no preferences button. I don't think the button is important enough to re-take all 10 screenshots (main, alts, and screen-specific for the eventual metadata service hook-up), so I'll go ahead and close.
